functions-methods.js

```diff
@@ -33,4 +56,18 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    if (email.includes("@") && email.lastIndexOf(".") !== email.length - 1 && email.indexOf(",") === -1) {
+        return "true";
+    } else {
+        return "false";
+    }
```

Comment:
Als je het als boolean zou willen teruggeven, dan zou je het nog iets kunnen versimpelen
```suggestion
    return email.includes("@") && email.lastIndexOf(".") !== email.length - 1 && email.indexOf(",") === -1;
```